index.html

```diff
@@ -24,15 +24,45 @@ <h1 class="title">Hi! I am Ezra nice to meet you.</h1>
                 If you like what you see and have a project you need coder, don't 
                 husitate to contact me.
             </p>
-            <h3 class="links">LET'S CONNECT</h3>
+        </main>
+    </section>
+    <section class="work-page">
+        <img src="images/works/portfolio1.svg" alt="work1" width="100%">
+        <h2 class="work-title">Tonic</h2>
+        <ul class="clients">
+            <li>CANOPY</li>
+            <img src="images/works/Counter.svg" alt="counter" class="counter">
+            <li>BackEndDev</li>
+            <img src="images/works/Counter.svg" alt="counter" class="counter">
+            <li>2015</li>
+        </ul>

```

Comment:
- [ ] Kindly space these elements correctly to match the Figma design
  
![image](https://user-images.githubusercontent.com/98453979/198717216-80342f1e-0f40-439f-a93b-f6da5eb6aad0.png)

### Checkout Figma
![image](https://user-images.githubusercontent.com/98453979/198717473-f1808618-bc92-4d22-b998-19cc347fe217.png)
 design

- [ ] I noticed you only had one project card. Kindly add three more project cards to match the Figma design